src/NQueenTest.java

```diff
@@ -0,0 +1,24 @@
+import static org.junit.Assert.*;
+
+import org.junit.*;
+public class NQueenTest
+{
+	NQueen obj;
+	@Before
+	public void initializeClass() throws Exception
+	{
+		obj=new NQueen();
+	}
+	@Test
+	public void NQueenProblemTest()
```

Comment:
Cover At least one positive and one negative cases for every method.

Reply:
done

Reply:
Still missing test method for positive flow like when the size of array is 4X4. And also missing test method for case when size of array is greater than 3X3 but no solution exist for them.

Reply:
Added positive test flow